packages/ui/components/form/date-range-picker/Calendar.tsx

```diff
@@ -9,19 +9,22 @@ import { classNames as cn } from "@calcom/lib";
 import { buttonClasses } from "../../button/Button";
 import { Icon } from "../../icon";
 
-export type CalendarProps = React.ComponentProps<typeof DayPicker>;
+export type CalendarProps = React.ComponentProps<typeof DayPicker> & {
+  disablePastDates?: boolean;
+};
 
 function Calendar({
   className,
   classNames,
   fromDate,
   toDate,
   showOutsideDays = true,
+  disablePastDates = true,
```

Comment:
would prefer this to be a minDate option (set by default to new Date()) - and nullable to allow all dates enabled.